app/Http/Middleware/CheckJWT.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Auth0\SDK\JWTVerifier;
```

Comment:
Note to everyone: 
`@deprecated 5.7.0, replacement available in upcoming 7.0.0: Auth0\SDK\Helpers\Tokens\IdTokenVerifier`